src/chainlit/client/utils.py

```diff
@@ -1,26 +1,43 @@
-from typing import Optional
+import json
+from typing import Dict, Optional
 
 from fastapi import Request
+from starlette.datastructures import Headers
 
 from chainlit.client.base import BaseAuthClient, BaseDBClient, UserDict
 from chainlit.client.cloud import CloudAuthClient, CloudDBClient
 from chainlit.client.local import LocalAuthClient, LocalDBClient
 from chainlit.config import config
-from chainlit.telemetry import trace_event
 
 
-async def get_auth_client(authorization: Optional[str]) -> BaseAuthClient:
-    # Check authorization
-    if not config.project.public and not authorization:
-        # Refuse connection if the app is private and no access token is provided
-        trace_event("no_access_token")
-        raise ConnectionRefusedError("No access token provided")
-    elif authorization and config.project.id:
+def load_chainlit_cookie(request: Request):
+    cookie_string = request.cookies.get("chainlit-headers")
```

Comment:
Where is this cookie set?

Reply:
Then you should use the same cookie name, right?

```suggestion
    cookie_string = request.cookies.get("chainlit-initial-headers")
```